runtime/include/jvmti.h

```diff
@@ -340,6 +340,9 @@ typedef enum jvmtiError {
 	JVMTI_ERROR_NAMES_DONT_MATCH = 69,
 	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_MODIFIERS_CHANGED = 70,
 	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_METHOD_MODIFIERS_CHANGED = 71,
+#if defined(J9VM_OPT_VALHALLA_NESTMATES)
+	JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_ATTRIBUTE_CHANGED = 72,
```

Comment:
Is there a new JVMTI spec that adds this error?

Reply:
Yes - mailing list link is here: http://mail.openjdk.java.net/pipermail/valhalla-spec-experts/2018-February/000541.html

In the web revision, it adds:
```
 JVMTI_ERROR_UNSUPPORTED_REDEFINITION_CLASS_ATTRIBUTE_CHANGED (72)
    A new class version has unsupported differences in class attributes.
```
(found under 'Function Specific Agent Errors' here: http://cr.openjdk.java.net/~dholmes/8010319/specs/JVM-TI/diff.html)

Reply:
OK